src/ostree/ot-builtin-static-delta.c

```diff
@@ -226,7 +226,7 @@ ot_static_delta_builtin_reindex (int argc, char **argv, OstreeCommandInvocation
   if (!ostree_option_context_parse (context, reindex_options, &argc, &argv, invocation, &repo, cancellable, error))
     return FALSE;
 
-  if (!ostree_cmd__private__ ()->ostree_static_delta_reindex (repo, opt_to_rev, cancellable, error))
+  if (!ostree_repo_static_delta_reindex (repo, 0, opt_to_rev, cancellable, error))
```

Comment:
Obviously if this is public API (which to me it should be), then I think you should reorder the commits so the `cmd_private__` API isn't ever added.

Reply:
Yeah, I didn't want to make the new changes to large to review though.

Reply:
New version is reordered